lib/db/users.ts

```diff
@@ -0,0 +1,25 @@
+import create_client from "./create_client";
+
+export type user = {
+  email: string;
+  password: string;
+};
+
+export const find_user = async (email: string): Promise<user | undefined> => {
+  const sql = `SELECT u.email,
+                            u.password
+                            FROM Users AS u
+                            WHERE u.email=$1`;
+  const client = create_client();
+  let user = undefined;
+  try {
+    await client.connect();
+    const res = await client.query(sql, [email]);
+    user = res.rows[0];
```

Comment:
Funkar det här? alltså vi vill ju ha user som undefined om den inte finns, eller kastar den någon exeception när det är index out of bounds?

Reply:
added a check